src/libraries/System.Drawing.Common/tests/AssemblyInfo.cs

```diff
@@ -5,3 +5,4 @@
 using Xunit;
 
 [assembly: ActiveIssue("https://github.com/dotnet/runtime/issues/35917", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))]
+[assembly: SkipOnMono("System.Drawing.Common is not supported on Browser", TestPlatforms.Browser)]
```

Comment:
Can you also add

```xml
<IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
```

This will prevent this test assembly to be published to helix and avoid running a no-op test assembly.

Reply:
Added